src/main/io/gps.c

```diff
@@ -826,6 +1309,19 @@ void gpsInitHardware(void)
     }
 }
 
+static void requestSatelliteInfo(void)
+{
+    if (gpsData.ubloxM8orAbove) {
```

Comment:
`ubloxM8orAbove`: the convenience method that made coding inconvenient :)

[Coding tip: avoid nesting like the plague](https://testing.googleblog.com/2017/06/code-health-reduce-nesting-reduce.html)

```suggestion
    if (gpsData.unitVersion >= UBX_VERSION_M9) {
        ubloxSetMessageRateValSet(CFG_MSGOUT_UBX_NAV_SAT_UART1, 5);
    } else if (gpsData.unitVersion == UBX_VERSION_M8) {
        ubloxSetMessageRate(CLASS_NAV, MSG_NAV_SAT, 5);
    } else {
        ubloxSetMessageRate(CLASS_NAV, MSG_NAV_SVINFO, 5); // for older models
    }
```

Reply:
This has been refactored, please comment on the changed code.